dxFileUploader: describe upload methods (#1200)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxFileUploader/3 Methods/upload().md

---
---
##### shortDescription

> Uploads all selected files.

Reviewer: ```suggestion
Uploads all the selected files.
```

---

[note]

> File upload is not supported in **useForms** upload mode ([uploadMode](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode)). In **instantly** upload mode, use [value](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#value) to select a file(s) for uploading.

Reviewer: ```suggestion
File upload is not supported in **useForms** upload mode ([uploadMode](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode)). In **instantly** upload mode, use [value](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#value) to select a file(s) you want to upload.
```
